.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
You still need to remove this file and then commit the change. `.gitignore` works on a "from here on we do this..." basis, meaning it will prevent future files that are ignored from being added to the commit. It will **not** remove already committed ones.